src/System.Runtime.Numerics/src/System/Numerics/BigIntegerCalculator.AddSub.cs

```diff
@@ -5,6 +5,68 @@ namespace System.Numerics
 {
     internal static partial class BigIntegerCalculator
     {
+        public static uint[] Add(uint left, uint right)
+        {
+            // Executes the trivial addition for two 32-bit integers.
+            // Since the caller is working with uint[] objects, we're
+            // doing him a favor with already providing it that way.
+
+            uint[] bits = new uint[2];
```

Comment:
The result may have more than 32 bits. Since the caller does already some fancy compression (ctor of `BigInteger`), there would be no benefit in doing it here too.